docs/i18n-languages/japanese/CONTRIBUTING.md

```diff
@@ -0,0 +1,143 @@
+<!-- do not translate this -->
+| [Read these guidelines in other languages](/docs/i18n-languages) |
+|-|
+<!-- do not translate this -->
+
+
+# コントリビューション ガイドライン
+
+こんにちは！ 👋
+
+freeCodeCamp.org が実現できたのは、あなたのような数千人のボランティアが協力してくれたおかげです！ 私たちはあなたのコントリビューションに感謝します。そして、あなたが仲間になってくれるのを歓迎します。
+
+私たちは参加者全員に対して、私たちの[「行動規範」("Code of Conduct")](https://www.freecodecamp.org/code-of-conduct)を守ることを厳格に求めています。時間をとって必ず読んでください。読むのには2分もかかりません。
+
+Happy contributing.
+
+## プロジェクトを助けることができる3つの方法
+
+次のどれかの方法でコントリビューションができます。
+
+1. [このオープンソースのコードベースにコントリビュートする](#contribute-to-this-open-source-codebase)。コーディングチャレンジの編集をしたり、学習プラットフォームのバグ修正をして助けてください。
+
+2. 他のキャンプ参加者を [public forum](https://www.freecodecamp.org/forum/) で助けてください。[他の参加者のコーディングに関する質問に答えたり](https://www.freecodecamp.org/forum/?max_posts=1)、[他の参加者のコーディングプロジェクトにフィードバックを返してあげてください](https://www.freecodecamp.org/forum/c/project-feedback?max_posts=1)。
+
+3. 私たちの [YouTube チャンネルの動画](https://www.youtube.com/channel/UC8butISFwT-Wl7EV0hUK0BQ/videos) の字幕 (closed captions) を作成するのを助けてください。
```

Comment:
This does not seem to be a direct translation of the current English version of this line (see below):
```
3. Help us add subtitles (closed captions) to our [YouTube channel videos](https://www.youtube.com/channel/UC8butISFwT-Wl7EV0hUK0BQ/videos).
```